templates/ghost/meta.yaml

```diff
@@ -43,8 +48,12 @@ schema:
     appServiceImage:
       type: string
       title: App Service Image
-      default: ghost:5.35.1-alpine
+      default: ghost:5.53.3-alpine
     databaseServiceName:
       type: string
       title: Database Service Name
       default: ghost-db
+    databaseType:
+      type: string
+      title: Database Type
+      default: mysql
```

Comment:
Do you want to support multiple dbs? Otherwise, this field is not useful.

Reply:
Ghost 5.0 only support MySQL 8 on Production,

https://ghost.org/docs/faq/supported-databases/

This option is indeed not usefull. I will remove it now.